gapic/src/main/com/google/gapid/views/CommandOptions.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.gapid.views;
+
+import static com.google.gapid.util.Paths.lastCommand;
+
+import com.google.gapid.models.CommandStream;
+import com.google.gapid.models.Models;
+import com.google.gapid.util.Experimental;
+import com.google.gapid.views.CommandTree.Tree;
+import com.google.gapid.widgets.Widgets;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Menu;
+import org.eclipse.swt.widgets.MenuItem;
+
+public class CommandOptions {
+  private final Menu optionsMenu;
+  private final Menu experimentsMenu;
+  private final Tree tree;
+  private final Widgets widgets;
+  private final Models models;
+
+  public CommandOptions(Control parent, Widgets widgets, Tree tree, Models models) {
+    this.optionsMenu = new Menu(parent);
+    this.experimentsMenu = new Menu(this.optionsMenu);
+    this.widgets = widgets;
+    this.tree = tree;
+    this.models = models;
+
+    MenuItem editMenuItem = Widgets.createMenuItem(this.optionsMenu , "&Edit", SWT.MOD1 + 'E', e -> {
+      CommandStream.Node node = this.tree.getSelection();
+      if (node != null && node.getData() != null && node.getCommand() != null) {
+        this.widgets .editor.showEditPopup(this.optionsMenu.getShell(), lastCommand(node.getData().getCommands()),
+            node.getCommand(), node.device);
+      }
+    });
+
+    if(!Experimental.enableProfileExperiments(this.models.settings)) {
```

Comment:
Still missing spaces after `if`. Here and throughout. Look at the Google Java style guide.

Reply:
Why don't we have a linter for these?